lib/main_common.pm

```diff
@@ -3003,13 +3003,12 @@ sub load_kernel_baremetal_tests {
     set_var('ADDONURL', 'sdk') if (is_sle('>=12') && is_sle('<15')) && !is_released;
     loadtest "kernel/ibtests_barriers" if get_var("IBTESTS");
     loadtest "autoyast/prepare_profile" if get_var("AUTOYAST_PREPARE_PROFILE");
+    load_boot_tests();
+    get_var("AUTOYAST") ? load_ayinst_tests() : load_inst_tests();
+    load_reboot_tests();
```

Comment:
I do not check too many details. It seems these 3 lines will be executed for IPXE case too. Is it needed?

Reply:
Yes, all those lines are need. Otherwise, there will be  no installation. It is only for kernel tests.